etc/inc/disable-common.inc

```diff
@@ -167,6 +167,10 @@ blacklist ${RUNUSER}/gnome-session-leader-fifo
 blacklist ${RUNUSER}/gnome-shell
 blacklist ${RUNUSER}/gsconnect
 
+# i3 IPC socket (allows arbitrary shell script execution)
+blacklist ${RUNUSER}/i3/ipc-socket.*
+blacklist /tmp/i3-*/ipc-socket.*
```

Comment:
Is there anything else in the i3 directory?

XY: Would it make sense to blacklist the directory?

Reply:
On my system, the only other files in there are an error log and another UNIX domain socket called `log-stream-socket.*`. Seems neither very dangerous nor very useful, so it probably doesn't matter either way.

Reply:
Logs could contain sensitive information.

If there is thing important in it we should blacklist the directory IMHO.

Reply:
FWIW, I agree. Targetting the directory is the better option here.

Reply:
Fair enough, I changed it to block the entire directories.